### PR TITLE
LPS-169174 Remove usages of MenuItem in sharing

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this about?

https://issues.liferay.com/issues/browse/LPS-169174

Replace usages of MenuItem with DropdownItem in sharing.

# How do I test it?

Share a document or blogs entry. Go to "Shared Content" and verify that all dropdown actions in both the blog entry and the document work.

This is an internal change, so no user visible changes should be noticeable other than markup changes.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1307401_5465

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sharing

**Comment by adolfopa** [comment]
@austinchiang Could these failures be caused by markup changes? I've replaced an old tag (`liferay-ui:menu`) with the current one (`clay:dropdown-actions`).

File: modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/display/context/ViewSharedAssetsDisplayContext.java [hunk 1]
```diff
@@ -219,65 +221,47 @@ public String getSharingEntryAssetTypeTitle(SharingEntry sharingEntry)
...
-		return menu;
+		return DropdownItemListBuilder.add(
+			() -> _hasEditPermission(
+				sharingEntry.getClassNameId(), sharingEntry.getClassPK()),
+			dropdownItem -> {
+				dropdownItem.setHref(
+					_getURLEdit(
+						sharingEntry, _liferayPortletRequest,
+						_liferayPortletResponse));
+				dropdownItem.setLabel(
```
Comment (AliciaGarciaGarcia) [inline modules/apps/sharing/sharing-web/src/main/java/com/liferay/sharing/web/internal/display/context/ViewSharedAssetsDisplayContext.java:241]: maybe add the icon?

or ark Maria about it?

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
ci:stop

**Comment by austinchiang** [comment]
**Reviewing...** :eyes:

**Comment by austinchiang** [comment]
@adolfopa you are correct, I need to update the tests

**Comment by austinchiang** [comment]
ci:test

**Comment by austinchiang** [comment]
ci:test

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by austinchiang** [comment]
There are more tests failing that was originally detected by ci:test:sharing, will need more time to investigated/fix the failures

**Comment by austinchiang** [comment]
ci:test

**Comment by austinchiang** [comment]
ci:test:echo

**Comment by adolfopa** [comment]
Making this a draft to make sure we don't forward it by mistake.

**Comment by austinchiang** [comment]
ci:test

**Comment by austinchiang** [comment]
ci:test:echo

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**